library/spdm_crypt_lib/libspdm_crypt_asym.c

```diff
@@ -979,10 +1004,12 @@ bool libspdm_asym_verify_ex(
         }
     }
     if (try_big_endian && try_little_endian && result) {
-        if (little_endian_succeeded) {
-            *endian = LIBSPDM_SPDM_10_11_VERIFY_SIGNATURE_ENDIAN_LITTLE_ONLY;
-        } else {
-            *endian = LIBSPDM_SPDM_10_11_VERIFY_SIGNATURE_ENDIAN_BIG_ONLY;
+        if (!libspdm_is_palindrome(signature, sig_size)) {
```

Comment:
This is for whole signature, right?

RSA is OK.

But ECDSA includes 2 parts - R and S.
Should we test R and S separately?

Reply:
You are correct. I forgot.

Reply:
@jyao1 I updated the changes for your review.